Content.Server/Atmos/EntitySystems/BarotraumaSystem.cs

```diff
@@ -287,5 +297,24 @@ public override void Update(float frameTime)
                 }
             }
         }
+
+        private void OnDamaged(EntityUid uid, BarotraumaComponent comp, DamageChangedEvent args)
+        {
+            // We actually only care that the entity with an equipped PressureProtectionComponent has been damaged. But
+            // since there's no (easy?) way to get the wearer of an item, and since we know that if the worn item is
+            // damaged then the wearer must be too, have the wearer subscribe to this event instead.
+            UpdateCachedResistances(uid, comp);
+        }
+
+        private void OnExamine(Entity<PressureProtectionComponent> ent, ref ExaminedEvent args)
```

Comment:
This event subscription should be moved to a shared system otherwise the examine box will mispredict. The data here isn't reliant on any atmos data so you should just make an `ESSharedBarotraumaSystem` in the `_ES` directory and put this method in there.

Reply:
Sure.

Reply:
`PressureProtectionComponent` is currently server only. Not sure I want to be sharing and networking components at this stage.

Reply:
very annoying but understandable